File: ARRAY/CPP/largest_number.cpp
#include<bits/stdc++.h>
using namespace std;

int largest(vector<int> nums){
    int n = nums.size();

    int maxi = INT_MIN;

    for(int i=0; i<n; i++){
        if(nums[i]>maxi){
            maxi = nums[i];
        }
    }
    return maxi;
}

int main(){
    int n;
    cout<<"Enter the size : ";
    cin>>n;

    vector<int> arr(n);

    for(int i=0; i<n; i++){
        cout<<"Enter the element : ";
        cin>>arr[i];
    }

    cout<<"The largest number in the array "<<largest(arr)<<endl;
}

File: ARRAY/PYTHON/largest_number.py
def largest(arr):
    n = len(arr)

    maxi = -111

    for i in range(n):
        if arr[i] > maxi:
            maxi = arr[i]

    return maxi

if __name__ == '__main__':
    user = input("Enter the array : ")
    list = [int(i) for i in user.split()]
    print(largest(list))